src/api/app/controllers/webui/packages/build_log_controller.rb

```diff
@@ -105,13 +105,17 @@ def check_build_log_access
         @package_name = params[:package]
 
         # No need to check for the package, they only exist on the backend in this case
+        # FIXME: scmsync is not the criteria, should also be done for remote packages
         if @project.scmsync
-          @can_modify = User.possibly_nobody.can_modify?(@project)
```

Comment:
`@can_modify` is used in the views to decide if we offer rebuild/abort build buttons.